packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -307,17 +307,17 @@ const Conversation = WebexPlugin.extend({
 
   /**
   * Gets an array of activities with an array of activity URLS
-  * @param {Array} activities
+  * @param {Array} activityUrls
   * @param {String} [cluster] cluster where the activities are located
   * @returns {Promise<Object>} Resolves with the activities
   * TODO: add cluster functionality when clusters are ready
   */
-  bulkActivitiesFetch(activities, cluster) {
+  bulkActivitiesFetch(activityUrls, cluster) {
     const resource = 'bulk_activities_fetch';
     const params = {
       method: 'POST',
       body: {
-        activities
+        activityUrls
```

Comment:
this was incorrect api call and service would return an error saying that we were not passing any activity urls